src/courtroom.cpp

```diff
@@ -1590,7 +1594,10 @@ void Courtroom::on_chat_return_pressed()
 
   packet_contents.append(ao_app->get_emote(current_char, current_emote));
 
-  packet_contents.append(ui_ic_chat_message->text());
+  QString text = ui_ic_chat_message->text();
+  if (text == "")
+    text = " "; // blankpost time
```

Comment:
Can we not fix this on the server side... \*head hitting wall\*

Reply:
Oh lol we could

Reply:
actually no the packet doesn't get properly sent unless I do this for some fucking reason @oldmud0, might need further investigation but for now this works